py-polars/tests/unit/io/test_ipc.py

```diff
@@ -266,3 +267,42 @@ def test_ipc_view_gc_14448() -> None:
     df.write_ipc(f, future=True)
     f.seek(0)
     assert_frame_equal(pl.read_ipc(f), df)
+
+
+@pytest.mark.slow()
+@pytest.mark.write_disk()
+@pytest.mark.parametrize("stream", [True, False])
+def test_read_ipc_only_loads_selected_columns(
```

Comment:
Can we sessionize (pytest context) the creation of these files? As we want to create them more often and they are relatively expensive compared to other tests.

Reply:
In this particular case we're writing two different ways, with `write_ipc()` and `write_ipc_stream()`, and only the latter triggered high memory due to the block-size issue. So it's not something we'd want to do for this particular test, we want to cover path paths.